Podfile.lock

```diff
@@ -68,4 +68,4 @@ SPEC CHECKSUMS:
 
 PODFILE CHECKSUM: c118a2a371f6c1c49231939b145ae4b52eefa18f
 
-COCOAPODS: 1.8.4
+COCOAPODS: 1.9.1
```

Comment:
I think this change is causing tests to fail, because it's a different version of CocoaPods than what is specified in https://github.com/wordpress-mobile/WordPressKit-iOS/blob/3f6f92b9a238e81551add3b2eae420a97a7b4e1d/Gemfile#L2.